src/main/java/com/box/sdk/BoxResource.java

```diff
@@ -1,5 +1,11 @@
 package com.box.sdk;
```

Comment:
Not sure why but I am getting a checkstyle error on this page:
CheckStylePluginException: CheckStyle threw an exception when scanning: BoxResource.java: Couldn't create Checker from Box SDK CheckStyle.
Could we perhaps isolate isolate where it is happening and turn checkstyle off for that portion of the code?